operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -0,0 +1,146 @@
+---
+auth_enabled: true
+chunk_store_config:
+  chunk_cache_config:
+    enable_fifocache: true
+    fifocache:
+      max_size_bytes: 500MB
+compactor:
+  compaction_interval: 2h
+  shared_store: s3
+  working_directory: {{ .StorageDirectory }}/compactor
+frontend:
+  tail_proxy_url: http://{{ .Querier.FQDN }}:{{ .Querier.Port }}
+  compress_responses: true
+  max_outstanding_per_tenant: 256
+  log_queries_longer_than: 5s
+frontend_worker:
+  frontend_address: {{ .FrontendWorker.FQDN }}:{{ .FrontendWorker.Port }}
+  grpc_client_config:
+    max_send_msg_size: 104857600
+  parallelism: {{ .QueryParallelism.Value }}
+ingester:
+  chunk_block_size: 262144
+  chunk_encoding: snappy
+  chunk_idle_period: 1h
+  chunk_retain_period: 30s
+  chunk_target_size: 1048576
```

Comment:
You can safely double this.

Reply:
Solved here: https://github.com/grafana/loki/pull/4975